content/academy/apify_platform/deploying/deploying.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Deploying
-description: Push your code from your local environment to the platform, or create a new actor on the console and integrate it with a Github repository to automatically rebuild any new changes.
+description: Push your local code to the platform, or create a new actor on the console and integrate it with a Github repo to automatically rebuild any new changes.
 menuWeight: 4
 paths:
 - apify-platform/deploying/deploying
```

Comment:
Please make a few touch-ups to make it clear that rebuilding on push is optional. Typically it's a good idea for beta versions, but not for latest, because you might easily break actors in production.

Also, it's GitHub with capital H